tests/test-sources/plugins/by-name/vim-suda/default.nix

```diff
@@ -0,0 +1,5 @@
+{
+  empty = {
+    plugins.vim-suda.enable = true;
+  };
```

Comment:
If we have settings and an example now, there should be corresponding test cases here.

In total we should have `empty`, `defaults` and `example`

- `defaults` should set all values explicitly to upstream's defaults
- - `example` should ideally matching the settingsExample in the plugin module